src/patternfly/sass-utilities/mixins.scss

```diff
@@ -126,6 +126,18 @@
   border: 0;
 }
 
+@mixin pf-u-screen-reader-full-size {
```

Comment:
LGTM, we could have this as a modifier, and potentially a fixed/absolute modifier. There may be a use case when you want something to be full size but be able to break out of its relatively positioned parent and relative to some other element instead - you could use `position: fixed` then, and add CSS that creates a stacking context to the parent you want it positioned relative to - https://codepen.io/mcoker/pen/XWZGapY

Then we could use `.pf-u-screen-reader.pf-m-full-size` (width/height 100%) and `.pf-u-screen-reader.pf-m-absolute` (position absolute) instead of unique classes for different variations.

Just a thought, I'm probably overthinking it 😂

Reply:
That sounds like a great idea!